editor/plugins/shader_editor_plugin.cpp

```diff
@@ -405,13 +411,11 @@ void ShaderEditorPlugin::drop_data_fw(const Point2 &p_point, const Variant &p_da
 
 		for (int i = 0; i < files.size(); i++) {
 			String file = files[i];
-			if (!ResourceLoader::exists(file, "Shader")) {
-				continue;
-			}
-
-			Ref<Resource> res = ResourceLoader::load(file);
-			if (res.is_valid()) {
-				edit(res.ptr());
+			if (ResourceLoader::exists(file, "Shader") || ResourceLoader::exists(file, "ShaderInclude")) {
+				Ref<Resource> res = ResourceLoader::load(file);
+				if (res.is_valid()) {
+					edit(res.ptr());
+				}
```

Comment:
There is no need to nest everything like that, this was fine before. Generally prefer to keep nesting to the minimum, it makes code harder to read.